please/please.py

```diff
@@ -11,172 +11,192 @@
-        return'Night'
+# function is never called
 
+# def get_time_of_day(x:int) -> str:
+#     if 4 < x <= 12:
+#         return "Morning"
+#     elif 12 < x <= 16:
+#         return "Afternoon"
+#     elif 16 < x <= 20:
+#         return "Evening"
+#     elif (20 < x <= 24) or (x <= 4):
+#         return "Night"
 
-def all_tasks_done():
-    completed_all_tasks = True
-    for task in config["tasks"]:
-        if task["done"] == False:
-            completed_all_tasks = False
-    return completed_all_tasks
 
+def all_tasks_done() -> bool:
+    return all(task["done"] for task in config["tasks"])
 
-@ app.command(short_help="Change name without resetting data")
-def callme(name: str):
+
+@app.command(short_help="Change name without resetting data")
+def callme(name: str) -> None:
     config["user_name"] = name
     write_config(config)
     center_print("\nThanks for letting me know your name!\n", "black on green")
 
 
-@ app.command(short_help='Add a Task')
-def add(task: str):
-    new_task = {
-        "name": task,
-        "done": False
-    }
+@app.command(short_help="Add a Task")
+def add(task: str) -> None:
+    new_task = {"name": task, "done": False}
     config["tasks"].append(new_task)
     write_config(config)
-    center_print(f"Added \"{task}\" to the list",
-                 "cyan1 on purple3")
+    center_print(f'Added "{task}" to the list', "cyan1 on purple3")
     print_tasks(config["tasks"])
 
 
-@ app.command(short_help='Deletes a Task')
-def delete(index: int):
+@app.command(short_help="Deletes a Task")
+def delete(index: int) -> None:
     index = index - 1
-    if index >= len(config["tasks"]):
+    if len(config["tasks"]) == 0:
+        center_print_wrap("Task list is empty", style="bright_red on bright_white")
+    elif not 0 <= index < len(config["tasks"]):
         center_print_wrap(
-            "Are you sure you gave me the correct number to delete?", "bright_red on bright_white")
-        return
-
-    if len(config["tasks"]) > 0:
+            "Are you sure you gave me the correct number to delete?",
+            "bright_red on bright_white",
+        )
+    else:
         deleted_task = config["tasks"][index]
         del config["tasks"][index]
         write_config(config)
         center_print(f"Deleted '{deleted_task['name']}'", "cyan1 on purple3")
         print_tasks(config["tasks"], True)
-    else:
-        center_print_wrap(
-            "Sorry, I've got no tasks to delete", style="bright_red on bright_white")
 
 
-@ app.command(short_help='Mark a task as done')
-def done(index: int):
+@app.command(short_help="Mark a task as done")
+def done(index: int) -> None:
     index = index - 1
-    if index >= len(config["tasks"]):
-        center_print_wrap(
-            "Are you sure you gave me the correct number to mark as done?", "bright_red on bright_white")
-        return
 
-    if len(config["tasks"]) > 0 and not all_tasks_done():
+    if len(config["tasks"]) == 0:
```

Comment:
The conditions aren't mutually exclusive, it's better not to use if/elif here (mutually exclusive conditions should be handled in their own if/elif statements), so would have to be reverted unfortunately.